scripts/merch.ts

```diff
@@ -1,79 +1,27 @@
+import {
+  Product,
+  Variant,
+  toProductColourMap,
+  ProductColour,
+  toProductSizeMap,
+  ProductSize,
+  Price,
+} from "data/types";
 import Stripe from "stripe";
 
-export enum ProductColour {
-  UNKNOWN = "Unknown",
-  BLACK = "Black",
-  CREAM = "Cream",
-  POWDER = "Powder",
-  GREY = "Grey",
-  PINE = "Pine",
-  NAVY = "Navy",
-}
+//! Change to toggle on/off merch
+export const isMerchActive = (): boolean => false;
```

Comment:
Could prob just export the constant `false` rather than the boolean function, but this will make it easier to change later if we want to use an environment variable etc